docs/testspec/src/12_pubkey_kem.rst

```diff
@@ -109,53 +109,59 @@ tests are in *src/tests/data/pubkey/frodokem_kat.vec*.
    |                        |    with the original private key. Expect a decryption failure.          |
    +------------------------+-------------------------------------------------------------------------+
 
-Kyber
-~~~~~
+ML-KEM
+~~~~~~
 
-The implementation is tested for correctness using the Known Answer Test vectors
-demanded by the NIST submission and provided by the reference implementation.
+The implementation is tested for correctness using Known Answer Test vectors
+generated using an implementation associated with the ACVP project.
 
 Additionally, Botan has implementation-specific test cases. Those ensure the
 interoperability of the algorithm when using Botan's generic API for public key
 algorithms. These test cases are equal for all public key schemes and are
 therefore not discussed in detail in this chapter.
 
-All kyber-specific test code can be found in :srcref:`src/tests/test_kyber.cpp`.
-Relevant test data vectors for the KAT tests are in
-*src/tests/data/pubkey/kyber\_\*.vec* where *\** is a placeholder for the
-algorithm parameters, namely *512*, *512_90s*, *768*, *768_90s*, *1024* and
-*1024_90s*.
+All ML-KEM-specific test code can be found in
+:srcref:`src/tests/test_kyber.cpp`.
+Relevant test data vectors for the ML-KEM KAT tests are in
+:srcref:`[src/tests/data/pubkey]/kyber_encodings.vec`,
+:srcref:`[src/tests/data/pubkey]/ml_kem.vec`,
+:srcref:`[src/tests/data/pubkey]/ml_kem_acvp_keygen.vec`, and
+:srcref:`[src/tests/data/pubkey]/ml_kem_acvp_encap.vec`. Additionally, the
+vectors for testing the still supported Kyber (NIST Round 3 submission)
+instances are in :srcref:`[src/tests/data/pubkey]/kyber_kat.vec`.
 
 .. table::
    :class: longtable
    :widths: 20 80
 
    +------------------------+-------------------------------------------------------------------------+
-   | **Test Case No.:**     | PKENC-KYBER-1                                                           |
+   | **Test Case No.:**     | PKENC-ML-KEM-1                                                          |
    +========================+=========================================================================+
    | **Type:**              | Known Answer Tests                                                      |
    +------------------------+-------------------------------------------------------------------------+
-   | **Description:**       | Uses the KAT vectors of Kyber's reference implementation as specified   |
-   |                        | in the NIST submission                                                  |
+   | **Description:**       | Uses the KAT vectors for ML-KEM.                                        |
```

Comment:
This uses the Kris Kwiatkowski test for ML-KEM. For Kyber it is still the reference KATs

Reply:
I think it suffices here to speak about KAT vectors in general for ML-KEM/Kyber. The source of the vectors is stated above. Which files are used is stated below.